connectors/byoei.py

```diff
@@ -91,6 +96,9 @@ def __init__(
         self.chunk_mem_size = chunk_mem_size * 1024 * 1024
         self.max_concurrent_bulks = max_concurrency
         self.bulk_tasks = ConcurrentTasks(max_concurrency=max_concurrency)
+        self.indexed_document_count = 0
+        self.indexed_document_volume = 0
+        self.deleted_document_count = 0
```

Comment:
Should we also have `deleted_document_volume`?

Reply:
We don't have it in ruby either, so if we want this then it's a different PR